contents/spotlight/startup-bugprove.md

```diff
@@ -0,0 +1,78 @@
+
+### 1. Choose the right platform to focus on
+
+There are lots of different platforms where influencers can congregate, including obvious candidates like TikTok and Instagram — but Dora has learned that YouTube has some inherent benefits for marketers. 
+
+“I’m not a big fan of newsletters or other sort of content where it’s read once and deleted,” says Dora. “One of the advantages of YouTube is that if you work with an influencer then the video is indexed content on their channel that leads to your site. We still see sign-ups coming from content that was created months ago. YouTube has a very long tail in that sense.”
+
+However, this long tail could make YouTube a poor fit for other types of products, so it’s important to experiment with different approaches and choose the channel that fits with your goals. 
+
+“Basically what you have to do is try two or three different channels for three or four months,” says Dora. “Try some content marketing, try some influencer marketing. Do you get the best response with audiences on LinkedIn, or YouTube? It’s always experimentation at the start.”
+
+> **Putting it into practice:** Use a tool such as PostHog to track where users are coming from (often [using UTMs](/tutorials/performance-marketing)) and think about what behavior you want to encourage. Then you can experiment with different channels and get a clear read on how each one performs.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zs86OYea8Wk?si=jxjFSBG0WjIJVEV9" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+### 2. Make room for authentic engagements
+
+Influencer marketing can be a tricky business due to the inherent friction between influencer and marketer. On the company side, you want influencers to say lots of nice things about your product and to give an effective, on-brand pitch. Yet influencers themselves also want to protect _their_ brand and don’t want to simply become company shills. 
+
+The best route, according to Dora, is to let go of the reins and remember that influencers know their own audience best. 
+
+“We had a funny moment that illustrates this,” says Dora. “We had an advertisement with [John Hammond](https://www.youtube.com/watch?app=desktop&v=zs86OYea8Wk) planned to go in the middle of a video. We watched, and it turned out great, but then the video continued and he decided to actually try Bugprove out.”
+
+“So, he goes on, tries the tool, and we were just hoping that it would work well. Then he uses dark mode and we just weren’t ready for that at that point. It looked bad and our poor product designer almost fainted. Luckily, John found it funny and switched back to light mode and it worked great from then on. It was an authentic moment, so we were happy with it in the end.”
+
+(Bugprove has since made improvements to dark mode.)
```

Comment:
Not yet, but on the roadmap to avoid such moments in the future :)

Reply:
```suggestion
(Bugprove is planning to make improvements to dark mode in the near future.)
```